convert/jenkinsjson/json/withDockerContainer.go

```diff
@@ -0,0 +1,51 @@
+package json
+
+import (
+	"encoding/json"
+	"log"
+	harness "github.com/drone/spec/dist/go"
+)
+
+func ConvertWithDockerContainer(node Node, variables map[string]string) *harness.Step {
```

Comment:
I don't think this is right. It should not return a step. Check OP's PR

Reply:
https://github.com/jamie-harness/go-convert/pull/23/files

Reply:
Really Sorry, I pushed withDockerContainer code by mistake and didn’t even realise. I'll revert back changes and for withDockerContainer, I am working on OP's branch.